### PR TITLE
My notes page with sidebar 

### PR DESCRIPTION
**Pull request by sushil-mod**
# Added MyNote page with sibar and other pages 
### how to check :-  
In Navbar click on login -> in login page click on login button -> you are now in mynotes page then click on sidebar links to navigate other pages 
## what to check 

- Sidebar :- src/components/sidebar/Sidebar.css
- Sidebar :- src/components/sidebar/Sidebar.jsx
-  MyNotes jsx , Archive jsx , Trash jsx , Label jsx ( mostly same 10 line of code )

## screenshot
homenote

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,21 +1,31 @@
 import MockMan from 'mockman-js';
 import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import Archive from './pages/archive/Archive';
 import Login from './pages/auth/Login';
 import Signup from './pages/auth/Signup';
 import Home from "./pages/home/Home";
+import Label from './pages/label/Label';
+import MyNotes from './pages/mynotes/MyNotes';
+import Trash from './pages/trash/Trash';
```
Comment (maitrakhatri) [inline src/App.js]: can use barrel imports here for simplicity

File: src/components/sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    const hamburgHandler = (e)=>{
+        e.preventDefault();
+        if(showSidebar===true){
+        setShowSidebar(false) ;
+        }else{
+            setShowSidebar(true) ;
+        }
+        
+    }
+    console.log(showSidebar);
```
Comment (maitrakhatri) [inline src/components/sidebar/Sidebar.jsx]: remove console logs before pushing the code

File: src/pages/archive/Archive.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import {  FaArchive } from 'react-icons/fa'
+import Navbar from '../../components/navbar/Navbar';
+import Sidebar from '../../components/sidebar/Sidebar';
```
Comment (maitrakhatri) [inline src/pages/archive/Archive.jsx]: can use barrel imports for all components

File: src/pages/mynotes/MyNotes.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+import NoteCard from '../../components/card/NoteCard';
+import Navbar from '../../components/navbar/Navbar';
+import Sidebar from '../../components/sidebar/Sidebar';
+import { CgNotes} from "react-icons/cg"
+
+function MyNotes() {
+
+  return<>
+  <Navbar/>
+  {/* d-flex wd-100 height-100 style={{ display:"flex " }} style={{ width:"100%", }} style={{ height:"100%" ,color:"#E6E6E6" }} */}
```
Comment (maitrakhatri) [inline src/pages/mynotes/MyNotes.jsx]: remove unnecessary comments before pushing

**Review by maitrakhatri — APPROVED** [review]
Code looks good.
Good folder structure.
Left some suggestions, give them a look

File: src/components/sidebar/Sidebar.jsx (hunk 2 above)
Comment (sushil-mod) [reply src/components/sidebar/Sidebar.jsx]: thanks for pointing out . done

File: src/App.js (hunk 1 above)
Comment (sushil-mod) [reply src/App.js]: yes will use thanks for suggesting

File: src/pages/mynotes/MyNotes.jsx (hunk 4 above)
Comment (sushil-mod) [reply src/pages/mynotes/MyNotes.jsx]: yes  for sure .

File: src/pages/mynotes/MyNotes.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  <div className='d-flex' >
+        <Sidebar/>
+
+          <div className='wd-100' >
+                  <div className='flex-center flex-col height-100 grey-shade' >
+                    <div> <CgNotes size={"10rem"} /> </div>
+                    <div> <h2>Notes you add appear here </h2> </div>
```
Comment (tusharkandpal) [inline src/pages/mynotes/MyNotes.jsx:11]: can directly use `<h2>` , please remove extra `<div>`

File: src/pages/label/Label.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Navbar from '../../components/navbar/Navbar';
+import Sidebar from '../../components/sidebar/Sidebar';
+import { MdLabel } from "react-icons/md" ;
+
+function Label() {
+  return <>
+  <Navbar/>
```
Comment (tusharkandpal) [inline src/pages/label/Label.jsx]: can keep `Navbar` out of routes, instead of importing in each component

**Review by tusharkandpal — APPROVED** [review]
well organized & hierarchy maintained.

Suggestion: try to avoid inline style as much as possible.

File: src/pages/mynotes/MyNotes.jsx (hunk 5 above)
Comment (sushil-mod) [reply src/pages/mynotes/MyNotes.jsx:11]: ohk will try this

File: src/pages/label/Label.jsx (hunk 6 above)
Comment (sushil-mod) [reply src/pages/label/Label.jsx]: yeah i also thought the same will try to  implement

**Pull request merged**